services/crypto_update.proto

```diff
@@ -171,8 +171,13 @@ message CryptoUpdateTransactionBody {
         VirtualAddress add = 20;
 
         /**
-         * The 20-byte EVM address of the virtual address that is being removed.
-         */
-        bytes remove = 21;
+        * The 20-byte EVM address of the virtual address that is being disabled and therefore removed from active use on the account.
+        */
+        bytes disable = 21;
```

Comment:
Since a disabled address stays in state, shouldn't this flag be moved to `VirtualAddress`? If queries only return active addresses how would users know which old, inactive addresses they need to delete?

Reply:
Yeah good suggestion. Will leave till we're ready to turn on disable